lib/snippets/publish-lerna-independent-packages

```diff
@@ -32,7 +36,7 @@ function distTag() {
 
 const command = [
   "node_modules/.bin/lerna publish",
-  "from-package",
+  useFromGit ? "from-git" : "from-package",
```

Comment:
Give repos the option to publish what was declared from `lerna version`. ie. no bloody surprises.